main.py

```diff
@@ -29,8 +64,10 @@ def setup(self):
         log.debug("flow-manager starting")
         self._flow_mods_sent = OrderedDict()
         self._flow_mods_sent_max_size = FLOWS_DICT_MAX_SIZE
-        self.cookie_exception_range = CONSISTENCY_COOKIE_EXCEPTION_RANGE
-        self.table_id_exception_range = CONSISTENCY_TABLE_ID_EXCEPTION_RANGE
+        if validate_input(CONSISTENCY_COOKIE_EXCEPTION_RANGE):
+            self.cookie_exception_range = CONSISTENCY_COOKIE_EXCEPTION_RANGE
+        if validate_input(CONSISTENCY_TABLE_ID_EXCEPTION_RANGE):
+            self.tab_id_exception_range = CONSISTENCY_TABLE_ID_EXCEPTION_RANGE
```

Comment:
So, if the exception list from 'settings.py' is not valid, the attributes "cookie_exception_range" and "table_id_exception_range" will not be created? I think this is a problem. I think also, the best solution it will create an empty list for each one. This will prevent the crash caused by a wrong type item (like item of string type) in those lists. If you agree, the warning messages raised by 'validate_input' should warn the user about the exception list is not in expected format and these list will be ignored.

Reply:
Resolved in c3ae171